src/main/java/com/spotify/github/v3/clients/GithubAppClient.java

```diff
@@ -69,8 +77,18 @@ public CompletableFuture<List<Installation>> getInstallations() {
    * @return a list of Installation
    */
   public CompletableFuture<Installation> getInstallation() {
+    return maybeRepo.map(repo-> github.request(
```

Comment:
minor: you could extract the inner logic to another method e.g `getRepoInstallation`.